app/src/lib/baseBranch/baseBranchService.ts

```diff
@@ -45,7 +45,7 @@ export class BaseBranchService {
 				return;
 			} else if (err.code === Code.ProjectsGitAuth) {
 				showError('Failed to authenticate', err);
-			} else {
+			} else if (action !== undefined) {
```

Comment:
This causes the frontend to not how errors when background fetches are performed.